### PR TITLE
Cleanup shellcheck warnings

### PR DESCRIPTION
**Pull request by vdbe**
Todo:
- [x] ./catppuccin.tmux
- [x] Modules
  - [x] ./pane
  - [x] ./status
  - [x] ./window
- [x] Update workflow to check warnings

local command used: `shellcheck -a catppuccin.tmux $(ls custom/*.sh pane/*.sh status/*.sh window/*.sh)`

### CODE REVIEW

File: catppuccin.tmux [hunk 1]
```diff
@@ -63,53 +72,61 @@ main() {
...
   setw window-status-format "$window_format"
 
-  local window_current_format=$(load_modules "window_current_format" "$modules_custom_path" "$modules_window_path")
+  window_current_format=$(load_modules "window_current_format" "$modules_custom_path" "$modules_window_path")
   setw window-status-current-format "$window_current_format"
 
   # status module
-  local status_left_separator=$(get_tmux_option "@catppuccin_status_left_separator" "")
-  local status_right_separator=$(get_tmux_option "@catppuccin_status_right_separator" "█")
-  local status_right_separator_inverse=$(get_tmux_option "@catppuccin_status_right_separator_inverse" "no")
```
Comment (vdbe) [inline catppuccin.tmux:103]: Removed unused option

File: pane/pane_default_format.sh [hunk 2]
```diff
@@ -1,12 +1,13 @@
 show_pane_default_format() {
-  local number="#{pane_index}"
-  local color="$(get_tmux_option "@catppuccin_pane_color" "$thm_green")"
-  local background="$(get_tmux_option "@catppuccin_pane_background_color" "$thm_gray")"
-  local text="$(get_tmux_option "@catppuccin_pane_default_text" "#{b:pane_current_path}")"
-  local fill="$(get_tmux_option "@catppuccin_pane_default_fill" "number")" # number, all, none
-  local active="#{pane_active}"
```
Comment (vdbe) [inline pane/pane_default_format.sh:7]: Removed unused option

**Comment by vdbe** [comment]
Example failed check: https://github.com/vdbe/catppuccin-tmux/actions/runs/8489882290/job/23260416298?pr=4

**Review by 89iuv — APPROVED** [review]

**Comment by 89iuv** [comment]
Nice work on cleaning up the code and setting up the github workflow.

File: .github/workflows/shellcheck.yml [hunk 3]
```diff
@@ -26,8 +26,5 @@ jobs:
       - uses: actions/checkout@v4
       - name: Run ShellCheck
         uses: ludeeus/action-shellcheck@master
-        with:
-          severity: error
-          additional_files: "catppuccin.tmux"
```
Comment (vdbe) [inline .github/workflows/shellcheck.yml:31]: `          additional_files: "catppuccin.tmux"`
This line should have been kept, my mistake.

**Pull request merged**